### DISCUSSION TITLE
Timer function

### DISCUSSION THREAD

**Discussion by Freiheitpt** [Koenkk/zigbee2mqtt#11640] [Devices]
Hello everyone,

Is it possible to make a device to turn off after X time has passed?
The idea is to turn on a ikea smart plug and after, for example, 3 minutes, turn it off.
Is it possible to do with something similar to binding? (I would have a switch, that switch would directly turn the smart pluig on and after 3 minutes the plug would turn off - all without intervention of zigbee2mqtt or any other software).

Thanks!

**Comment by lux73** [comment]
AFAIK no - this is for what your Home Automation Software is
